.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+
+python:
+    - 2.7
+
+env: DISPLAY=:99
+
+before_install:
+    - sudo apt-get update
+    - sh -e /etc/init.d/xvfb start
+
+install:
+    - sudo apt-get install flashplugin-nonfree
+    - pip install mozdownload pep8
+    - python setup.py develop
```

Comment:
Sounds like a good idea. Today is the beginning of my experience with travis, but this all seems pretty foolproof.